Example/Firebase.xcodeproj/project.pbxproj

```diff
@@ -65,6 +65,10 @@
 /* End PBXAggregateTarget section */
 
 /* Begin PBXBuildFile section */
```

Comment:
Run pod deintegrate

Reply:
Ahh, thanks! I was wondering if this was expected or not.